EducationPortal.BLL/DTO/ArticleDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace EducationPortal.BLL.DTO
+{
+    public class ArticleDTO : MaterialDTO
+    {
+        public DateTime Published { get; set; }
+
+        public new string MaterialType { get; } = "Article";
```

Comment:
Why do you use new for MaterialType? Use abstract for base property in the base abstract class and override it in a derived  class

Reply:
Updated to abstract property